usb/device/USBSerial/USBCDC.cpp

```diff
@@ -210,6 +210,18 @@ void USBCDC::callback_set_configuration(uint8_t configuration)
         _rx_in_progress = true;
 
         ret = true;
+
+        endpoint_remove_all();
+
+        endpoint_add(_int_in, CDC_MAX_PACKET_SIZE, USB_EP_TYPE_INT);
```

Comment:
@c1728p9 I notice now that this you mistakenly duplicated the code here

Reply:
Good catch. I added this to make sure endpoint remove worked correctly, but is ended up slipping into the PR. I have removed it.